docs/error-handling.md

```diff
@@ -52,7 +52,7 @@ app.use(async (ctx, next) => {
 
   Error event listeners can be specified with `app.on('error')`.
   If no error listener is specified, a default error listener
-  is used. Error listener receive all errors that make their
+  is used. Error listener receives all errors that make their
```

Comment:
updated with the changes

Reply:
Thank you!  Any chance you could pull in master branch and rebase?  I added myself to list of people receiving vulnerability notifications as @dead-horse mentioned, because I would love to continue to help make sure Koa sticks around for the long term.

Reply:
Sure I will rebase and then push the changes. @niftylettuce